cli/standalone.rs

```diff
@@ -137,12 +137,16 @@ impl ModuleLoader for EmbeddedModuleLoader {
     referrer: &str,
     _kind: ResolutionKind,
   ) -> Result<ModuleSpecifier, AnyError> {
+    // TODO(bartlomieju): ideally we shouldn't need to call `current_dir()` on each
+    // call
+    let cwd = std::env::current_dir().context("Unable to get CWD")?;
+
     // Try to follow redirects when resolving.
     let referrer = match self.eszip.get_module(referrer) {
       Some(eszip::Module { ref specifier, .. }) => {
-        deno_core::resolve_url_or_path_deprecated(specifier)?
+        deno_core::resolve_url_or_path(specifier, &cwd)?
```

Comment:
I'm not sure why `specifier` is not a `ModuleSpecifier`. We can just do `ModuleSpecifier::parse` here. https://github.com/denoland/eszip/issues/119

Reply:
That doesn't appear to be the case - I changed the code and tests started failing. It appears a referrer is `.`. I'm gonna leave this for a follow up.